src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/NomsIdSequence.java

```diff
@@ -0,0 +1,211 @@
+package uk.gov.justice.hmpps.prison.repository.jpa.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.apache.commons.lang3.StringUtils;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+@AllArgsConstructor
```

Comment:
do you need the all args if you've specified a builder?

Reply:
Resolved